cmake/OpenCVDetectCUDAUtils.cmake

```diff
@@ -109,7 +109,7 @@ macro(ocv_initialize_nvidia_device_generations)
   set(_arch_ampere   "8.0;8.6")
   set(_arch_lovelace "8.9")
   set(_arch_hopper   "9.0")
-  set(_arch_blackwell "10.0;12.0")
+  set(_arch_blackwell "10.0;10.3,11.0;12.0;12.1")
```

Comment:
Hm.. each option here is yet another kernel build. Longer build and fatter binary. I would say, that we do not need all possible combinations, if the target is defined by the arch name. "12.0;12.1" most probably should be just 12.1. The same for 10.x variations, if pure 10.0 devices do not exist and 10.3 is minimal production version.

Reply:
There is new option that is putting f 10.0f and takes optimizarion for all blackwell family

Reply:
> @johnnynunez Do you have any official documentation mentioning the new compute capabilities and the f option?

https://developer.nvidia.com/blog/nvidia-blackwell-and-nvidia-cuda-12-9-introduce-family-specific-architecture-features/